profiles/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from django.contrib.auth.hashers import check_password
+from django.contrib.auth.models import User
+from django.conf import settings
+from pdpdmeetup.settings import MEDIA_URL
+# Create your models here.
+import datetime    
+
+year_choices = []
+for r in range(1980, (datetime.datetime.now().year+1)):
+    year_choices.append((r,r))
+
+class Profile(models.Model):
+	user = models.OneToOneField(User, on_delete=models.CASCADE, null=True, blank=True)
+	name = models.CharField(null=True, blank=True, max_length=100)
```

Comment:
CharFields should never be nullable. Same goes with URLField. I would generally avoid nullable fields, unless really necessary. I think that all the fields here can be NOT nullable. This will lead to the rest of the code base with cleaner code, because there will be no need for any if statements to account for the field possibly being a None value instead of an integer/string/etc.

Also isn't this model suppose be the custom auth model?